app/packages/dataset/package.json

```diff
@@ -19,21 +19,13 @@
         "README.md"
     ],
     "dependencies": {
-        "@fiftyone/core": "*"
+        "@fiftyone/core": "*",
+        "framer-motion": "^6.2.8"
```

Comment:
nit: is this a peer dependency for one of the libs, because I don't see it being imported anywhere

Reply:
Hmm, I'm not sure it's showing this change. I don't see it locally.